src/components/inline_edit/inline_edit_form.test.tsx

```diff
@@ -524,4 +524,77 @@ describe('EuiInlineEditForm', () => {
       });
     });
   });
+
+  describe('inline edit as a controlled component', () => {
+    const onSave = jest.fn();
+    const onChange = jest.fn();
+    const onCancel = jest.fn();
+
+    beforeEach(() => {
+      onSave.mockReset();
+      onChange.mockReset();
+      onCancel.mockReset();
+    });
+
+    const controlledInlineEditFormProps: EuiInlineEditFormProps = {
+      ...requiredProps,
+      value: 'Hello World!',
+      onChange: onChange,
+      onCancel: onCancel,
+      inputAriaLabel: 'Edit inline',
+      sizes: MEDIUM_SIZE_FORM,
+      children: (readModeValue) => readModeValue,
+    };
+
+    it('renders in read mode', () => {
```

Comment:
I'd like to see the test name be more specific as well
```suggestion
    it('renders the passed value in read mode', () => {
```